app/client/components/helpCentre/helpCentreArticle.tsx

```diff
@@ -136,4 +146,103 @@ const ArticleBody = (props: ArticleBodyProps) => {
   return <div>{parseBody(props.article.body)}</div>;
 };
 
+const articleFeedbackWidgetCss = css`
+  display: flex;
+  flex-direction: column;
+  border: 1px solid ${neutral[86]};
+  padding: ${space[4]}px ${space[3]}px;
+  margin: 66px 0;
+  ${minWidth.mobileLandscape} {
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+  & p {
+    margin: 0;
+    ${textSans.medium({ fontWeight: "bold" })}
+  }
+  & .buttonDiv {
+    min-height: 36px;
+    display: flex;
+    align-items: center;
+    margin-top: ${space[4]}px;
+    ${minWidth.mobileLandscape} {
+      margin-top: 0;
+    }
+    & > * {
+      margin-right: ${space[2]}px;
+      ${minWidth.mobileLandscape} {
+        margin-right: ${space[3]}px;
+      }
+    }
+    & p {
+      ${textSans.small({ fontWeight: "regular" })}
+    }
+  }
+`;
+
+interface ArticleFeedbackWidgetProps {
+  articleCode: string;
+}
+
+export const ArticleFeedbackWidget = (props: ArticleFeedbackWidgetProps) => {
+  const [feedBackButtonClicked, setFeedBackButtonClicked] = useState(false);
+
+  return (
+    <div css={articleFeedbackWidgetCss}>
+      <p>Did you find the information you need?</p>
+      <div className="buttonDiv">
+        {feedBackButtonClicked ? (
+          <p>Thank you!</p>
+        ) : (
+          <>
+            <Button
+              icon={<ThumbsUpIcon />}
+              hideLabel={true}
+              size="small"
+              cssOverrides={css`
+                svg {
+                  width: initial;
+                }
+              `}
+              onClick={() => {
+                setFeedBackButtonClicked(true);
+                trackEvent({
+                  eventCategory: "help-centre-article-page",
+                  eventAction: "article-feedback-widget-click",
+                  eventLabel: props.articleCode,
+                  eventValue: 1
+                });
+              }}
+            >
+              Yes
+            </Button>
+            <Button
+              icon={<ThumbsUpIcon invertIcon={true} />}
+              hideLabel={true}
+              size="small"
+              cssOverrides={css`
+                svg {
+                  width: initial;
+                }
+              `}
+              onClick={() => {
+                setFeedBackButtonClicked(true);
+                trackEvent({
+                  eventCategory: "help-centre-article-page",
+                  eventAction: "article-feedback-widget-click",
+                  eventLabel: props.articleCode,
+                  eventValue: 0
```

Comment:
Same as above.

The `eventCategory` for the Help Centre entries needs to be `help-centre` as we do elsewhere in the Help Centre so its categorised correctly in GA and Ophan.

If you wanted to it would be safe to simplify `eventAction` to `article-feedback` as 'widget' and 'click' don't provide any addition info, given this is the only piece of article feedback being tracked and the only way to interact with it is clicking.